### PR TITLE
Fixed Broken/Outdated website Link

### PR DESCRIPTION
**Pull request by uzair-ali10**
On Line Number 233 
Before:- `https://www.tensorflow.org/guide/custom_estimators`
After:- `https://www.tensorflow.org/guide/estimator#custom_estimators`
`https://www.tensorflow.org/guide/custom_estimators`  is not a valid URL

### CODE REVIEW

**Comment by uzair-ali10** [comment]
@8bitmp3  @lamberta 
Do u have any specif format for jupyter notebooks to maintain?
If yes kindly send the docs link or reference for the same so that I can have a look at see what's the matter in my PR

**Comment by 8bitmp3** [comment]
Thank you. In this TF [blog post](https://blog.tensorflow.org/2020/10/how-tensorflow-docs-uses-juypter-notebooks.html) - go to "TensorFlow docs notebook tools" to learn about `nblint` and `nbfmt` and how they can help automate/identify fixes/issues with CI tests. Here's also the tools' repo with further instructions:  https://github.com/tensorflow/docs/tree/master/tools/tensorflow_docs/tools . And If you get stuck. let us know and we can implement the CI fixes for you.

**Comment by 8bitmp3** [comment]
If you click on "Details" next to "CI/Notebook lint (pull_request)" at the bottom of this page (https://github.com/tensorflow/docs/pull/1913), you should see that this lint test is notpassing because:

```
Fail | google::second_person | Prefer second person instead of first person: https://developers.google.com/style/person
```

You can address that by checking for "our", "we", etc and replacing them with "you", "your".

To pass the format test, you usually just run the `nbfmt` tool to clean some of the metadata with `python3 -m tensorflow_docs.tools.nbfmt notebook.ipynb` (more details https://github.com/tensorflow/docs/tree/master/tools/tensorflow_docs/tools).

**Comment by uzair-ali10** [comment]
@8bitmp3 
yes I've downloaded the notebook on my local machine and using `nblint` to make the necessary changes

**Comment by 8bitmp3** [comment]
Thanks. Once you fix the grammar and change the other parts to "it is recommended" and "the compatibility guarantees" (as mentioned above), you can format the notebooks with `python3 -m tensorflow_docs.tools.nbfmt your-notebook.ipynb` as described in the README https://github.com/tensorflow/docs/tree/master/tools/tensorflow_docs/tools - that should clean up the metadata in the Jupyter notebook (which is like a JSON file). Then, you can commit the changes and push them to your remote branch here.

File: site/en/tutorials/estimator/premade.ipynb [hunk 1]
```diff
@@ -230,8 +230,8 @@
         "`tf.estimator`\n",
         "(for example, `LinearRegressor`) to implement common ML algorithms. Beyond\n",
         "those, you may write your own\n",
-        "[custom Estimators](https://www.tensorflow.org/guide/custom_estimators).\n",
-        "We recommend using pre-made Estimators when just getting started.\n",
+        "[custom Estimators](https://www.tensorflow.org/guide/estimator#custom_estimators).\n",
+        "It is recommend using pre-made Estimators when just getting started.\n",
```
Comment (8bitmp3) [inline site/en/tutorials/estimator/premade.ipynb:234]:
```suggestion
        "It is recommended using pre-made Estimators when just getting started.\n",
```

File: site/en/tutorials/estimator/premade.ipynb [hunk 2]
```diff
@@ -295,7 +295,7 @@
       },
       "source": [
         "Your input function may generate the `features` dictionary and `label` list any\n",
-        "way you like. However, we recommend using TensorFlow's [Dataset API](https://www.tensorflow.org/guide/datasets), which can\n",
+        "way you like. However, It is recommend using TensorFlow's [Dataset API](https://www.tensorflow.org/guide/datasets), which can\n",
```
Comment (8bitmp3) [inline site/en/tutorials/estimator/premade.ipynb:298]:
```suggestion
        "way you like. However, It is recommended using TensorFlow's [Dataset API](https://www.tensorflow.org/guide/datasets), which can\n",
```

File: site/en/tutorials/estimator/premade.ipynb [hunk 3]
```diff
@@ -368,7 +368,7 @@
         "id": "jpKkhMoZljco"
       },
       "source": [
-        "Feature columns can be far more sophisticated than those we're showing here.  You can read more about Feature Columns in [this guide](https://www.tensorflow.org/guide/feature_columns).\n",
+        "Feature columns can be far more sophisticated than those which are being showing over here.  You can read more about Feature Columns in [this guide](https://www.tensorflow.org/guide/feature_columns).\n",
```
Comment (8bitmp3) [inline site/en/tutorials/estimator/premade.ipynb:371]:
```suggestion
        "Feature columns can be far more sophisticated than those shown here.  You can read more about Feature Columns in [this guide](https://www.tensorflow.org/guide/feature_columns).\n",
```

File: site/en/tutorials/estimator/premade.ipynb [hunk 4]
```diff
@@ -81,7 +81,7 @@
         "how to solve the Iris classification problem in TensorFlow using Estimators. An Estimator is a legacy TensorFlow high-level representation of a complete model. For more details see\n",
         "[Estimators](https://www.tensorflow.org/guide/estimator).\n",
         "\n",
-        "Note:  In TensorFlow 2.0, the [Keras API](https://www.tensorflow.org/guide/keras) can accomplish these same tasks, and is believed to be an easier API to learn. If you are starting fresh, we would recommend you start with Keras.\n"
+        "Note:  In TensorFlow 2.0, the [Keras API](https://www.tensorflow.org/guide/keras) can accomplish these same tasks, and is believed to be an easier API to learn. If you are starting fresh, it is recommend you start with Keras.\n"
```
Comment (8bitmp3) [inline site/en/tutorials/estimator/premade.ipynb:84]:
```suggestion
        "Note:  In TensorFlow 2.0, the [Keras API](https://www.tensorflow.org/guide/keras) can accomplish these same tasks, and is believed to be an easier API to learn. If you are starting fresh, it is recommended you start with Keras.\n"
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Grammar fixes - please take a look

File: site/en/tutorials/estimator/premade.ipynb (hunk 1 above)
Comment (uzair-ali10) [reply site/en/tutorials/estimator/premade.ipynb:234]: Yep I'm on it, Haven't finished my commit yet, Will inform you when I'm done :)

**Comment by uzair-ali10** [comment]
@8bitmp3 
Well, This is weird when I try to format with `nbfmt` it works fine but when I try to do the same with `nblint` 3 checks are failing, I've attached a screenshot below
Screenshot from 2021-07-06 22-50-57

From what I can understand it says the Colab, Github, and download icon shields do not have a correct URL Linked with them but that's not the case, All the URLs are working fine.
:(

**Comment by uzair-ali10** [comment]
@8bitmp3 
Cheers 🍻
Worked Like Charm :)

**Comment by 8bitmp3** [comment]
The recent rebase may have introduced changes to **105** files in this PR, can you please fix that.

**Comment by uzair-ali10** [comment]
@8bitmp3 
Yes, I'm aware of that, it is made by imgbot which is an Open-Source tool that optimizes and reduces the size of images that are present in the repository, All those extra 105 changes are made by imgbot which are basically optimized images
U can know more about imgbot from [here](https://imgbot.net/)
if u want I can revert the changes made by imgbot

**Comment by 8bitmp3** [comment]
Please only makes changes to the original notebook (1 file)

**Comment by uzair-ali10** [comment]
@8bitmp3 
Done!!
Check it out now

**Comment by 8bitmp3** [comment]
To tidy up the PR, you could also try cleaning the number of commits with `git rebase --interactive` e.g. `git rebase -i HEAD~30` (where 30 is the last 30 commits) to remove any "ImgBot" commits with `d` (where you were adding https://github.com/tensorflow/docs/pull/1913/commits/e27d5eced2e3a0e67485c0bdc2445acd512531f6 and removing unnecessary changes to ~150 files) 

You can learn about it in: 
- https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase#:~:text=Git%20Rebase%20Standard%20vs%20Git,accepts%20an%20--%20i%20argument.&text=Git%20rebase%20interactive%20is (Git Rebase Standard vs Git Rebase Interactive)
- https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by songzy12** [comment]
As a side note, the intended fix in this pull request is the same as https://github.com/tensorflow/docs/pull/1879/files right?

**Comment by uzair-ali10** [comment]
@songzy12 
Yes looks the same with a few more minor changes

**Comment by uzair-ali10** [comment]
@8bitmp3 
Or better shall download the `premade.ipynb` notebook and close this PR, Create a new nice and clean PR and make necessary changes? 
This PR looks like mess

**Comment by 8bitmp3** [comment]
Up to you. And thanks for the efforts.

**Comment by uzair-ali10** [comment]
@8bitmp3 
#1914 Here u go

**Comment by 8bitmp3** [comment]
Thanks, I will close this PR since https://github.com/tensorflow/docs/pull/1914 replaces it

**Pull request closed without merging**